### PR TITLE
Cleanups

### PR DESCRIPTION
**Pull request by aberaud**

### CODE REVIEW

File: include/opendht/sockaddr.h [hunk 1]
```diff
@@ -68,7 +68,7 @@ class OPENDHT_PUBLIC SockAddr {
      * Build from existing address.
      */
     SockAddr(const sockaddr* sa, socklen_t length) {
-        if (length > sizeof(sockaddr_storage))
+        if (length > (socklen_t)sizeof(sockaddr_storage))
```
Comment (AmarOk1412) [inline include/opendht/sockaddr.h:71]: oh a C-style cast

File: src/dhtrunner.cpp [hunk 2]
```diff
@@ -119,22 +119,24 @@ DhtRunner::run(const Config& config, Context&& context)
...
-                if (context.logger)
-                    context.logger->d("[runner %p] Using IPv4 port %hu from saved configuration", this, port);
-                if (local4.getPort() == 0)
+                if (local4.getPort() == 0) {
+                    if (context.logger)
+                        context.logger->d("[runner %p] Using IPv4 port %hu from saved configuration", this, port);
                     local4.setPort(port);
+                }
             }
             if (inConfig >> port) {
```
Comment (AmarOk1412) [inline src/dhtrunner.cpp:134]: In fact we can do this instruction only if local6.getPort() == 0

File: include/opendht/sockaddr.h (hunk 1 above)
Comment (aberaud) [reply include/opendht/sockaddr.h:71]: C-style casts are the best ;)

File: src/dhtrunner.cpp (hunk 2 above)
Comment (aberaud) [reply src/dhtrunner.cpp:134]: Indeed, but keeping like that since it's an uncommon case, and it's cleaner to have the same logic for both.

**Pull request merged**